modules/cbmongodb/models/Mongo/Collection.cfc

```diff
@@ -109,7 +109,7 @@ component name="MongoCollection" accessors="true"{
 	* @return mixed result|null 	Returns the result if found or returns null if the document was not found
 	*/
 	public function findById(required id){
-		var qId = getMongoUtil().newIDCriteriaObject(ARGUMENTS.id);
+		var qId = getMongoUtil().newIDCriteriaObject(arguments.id);
```

Comment:
I'm going to merge this, but I've started using uppercase for CF implicit scopes in the past year.  It makes it easier to scan the code for scoping issues, because most editors and code parsing utils use a different color for "constants"

If you see an update where I make a change all implicit scopes to use Ucase, don't be surprised. :)